content/applications/websites/website/configuration/multi_website.rst

```diff
@@ -2,316 +2,164 @@
 Multiple websites
 =================
 
-.. image:: multi_website/multi_website04.png
-  :align: center
+Odoo allows you to create multiple websites from the same database. This can be useful, for example,
+if you have multiple brands operating under your organization, or to create separate websites for
+different products/services, or different audiences. In these cases, having different websites can
+help avoid confusion and make it easier to tailor your digital outreach strategies and reach your
+target audience.
+
+Each website can be designed and configured independently with its own :doc:`domain name
+<../../../../administration/maintain/domain_names>`, theme, pages,
```

Comment:
you can write an "absolute path" (not really absolute but....) since you go back to the root folder anyway. to do so, it starts with `/[folder]`
```suggestion
</administration/maintain/domain_names>`, theme, pages,
```